Example-iOS/DemoStrings.swift

```diff
@@ -30,7 +30,10 @@ enum DemoStrings {
     static let xmlExample: NSAttributedString = {
 
         // This would typically come from NSLocalizedString
-        let localizedString = "I want to be different. If everyone is wearing <black><BON:noBreakSpace/>black,<BON:noBreakSpace/></black> I want to be wearing <red><BON:noBreakSpace/>red.<BON:noBreakSpace/></red>\n<signed><BON:emDash/>Maria Sharapova</signed> <racket/>"
+        let localizedString = """
+            I want to be different. If everyone is wearing <black><BON:noBreakSpace/>black,<BON:noBreakSpace/></black> I want to be wearing <red><BON:noBreakSpace/>red.<BON:noBreakSpace/></red>
```

Comment:
I could have done some soft wrapping here, but with all the slashes already, I thought it made things more confusing, not less.